Doc/library/mmap.rst

```diff
@@ -232,6 +232,21 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
       .. versionadded:: 3.8
 
 
+   .. method:: mprotect(prot[, start[, length]])
```

Comment:
The signature of this method needs discussion first, in particular the start and length parameters.

The code in mmapmodule.c currently assumes that the access mode is the same for the entire buffer. That means we can't change the access mode/protection for part of the buffer without rework for the rest of the module.